HerPortal.ManagementShell/AdminAction.cs

```diff
@@ -113,8 +114,38 @@ public void RemoveConsortia(User user, IReadOnlyCollection<string> consortiumCod
                 .Contains(code)
             ).ToList();
         if (missingCodes.Count > 0)
-            throw new CouldNotFindAuthorityException("Consortium Codes are not associated with this user.", missingCodes);
+            throw new CouldNotFindAuthorityException("Consortium Codes are not associated with this user.",
+                missingCodes);
 
         dbOperation.RemoveConsortiaFromUser(user, consortiaToRemove);
     }
+
+    public List<User> GetUsers()
+    {
+        return dbOperation.GetUsersWithLocalAuthoritiesAndConsortia();
+    }
+
+    public void MigrateAdmin(User user)
+    {
+        var consortiumCodesToAdd = GetConsortiumCodesUserShouldOwn(user).ToList();
+        var custodianCodesToRemove = GetOwnedCustodianCodesInConsortia(user, consortiumCodesToAdd);
```

Comment:
in this case I can only remove LAs that the user also owns, so at some point I'd need to filter by ones the user owns

on transactions, this could perhaps be done here, however the other commands also don't start a transaction until user confirmation has been given, and only includes writes to the database

Reply:
thinking about it a bit more I see what you mean now, they will own all those LAs by definition, will change